PyRoute/Galaxy.py

```diff
@@ -488,3 +488,7 @@ def process_owned_worlds(self):
                 g.write(ow_list_world + '\n')
 
                 world.ownedBy = (owner, ownedBy[0:4])
+
+    def is_well_formed(self):
```

Comment:
I ended up discovering (by tripping over) implicit requirements in the codebase for Star objects to be well-formed.

Rather than leave those requirements implicit, I've broken them out into an is_well_formed method, making it easier for later requirements to be plugged in.

Likewise, a if a single Star object in a Galaxy is _not_ well-formed, then the containing Galaxy is not well formed.

Reply:
On other hand, the canonical names for a representation-invariant checker are repOK (or rep_ok) and isOK (or is_ok) - would it be less confusing, @tjoneslo , to rename is_well_formed to one of those?  If so, which one?

Reply:
`is_well_formed()` seems perfectly sensible to me.